modules/ui_patterns_library/templates/patterns-meta-information.html.twig

```diff
@@ -7,30 +7,51 @@
 
 {% if pattern is not empty %}
 
-  {# Pattern name and desciption. #}
+  {# Pattern name and description. #}
   <h3 class="pattern-preview__label">{{ pattern.label }}</h3>
   <p class="pattern-preview__description">{{ pattern.description }}</p>
 
   {# Pattern fields descriptions. #}
-  <table class="pattern-preview__fields">
-    <thead>
-    <tr>
-      <th>{{ "Field"|t }}</th>
-      <th>{{ "Label"|t }}</th>
-      <th>{{ "Type"|t }}</th>
-      <th>{{ "Description"|t }}</th>
-    </tr>
-    </thead>
-    <tbody>
-    {% for field in pattern.fields %}
+  {% if pattern.fields or pattern.additional.settings %}
```

Comment:
Can't we do this with twig blocks? So that we are not opinionated about any specific data structure in the core module. The UI Patterns Settings can then extend this template and add the necessary code using the block defined here.

Reply:
We can, but I think in any case, it will be possible to extend only one time this logic of Twig blocks (in UI Patterns Settings).

By example, if another module adds a new structure then impossible for both modules to extend this pattern.

Also in the themes like ui_suite_bootstrap (ui_suite_zurb_foundation later) we override this template to add some styling https://git.drupalcode.org/project/ui_suite_bootstrap/-/blob/1.0.x/templates/overrides/ui_patterns_library/patterns-meta-information.html.twig.

So I am not sure if it is worth the effort.

If you still want me to do it with Twig blocks, then ok, I will do that.